hardware_interface/source/skytraq_binary.c

```diff
@@ -378,13 +402,14 @@ GPS_RETURNSTATE skytraq_configure_power_mode(skytraq_power_mode mode, skytraq_up
     return skytraq_send_message(payload, length);
 }
 
-GPS_RETURNSTATE skytraq_get_gps_time(uint8_t *reply) {
+GPS_RETURNSTATE skytraq_get_gps_time(uint8_t *time) {
     uint16_t length = 2;
     uint8_t payload[2];
+    uint8_t reply[100];
 
     *(uint16_t *)&(payload[0]) = QUERY_GPS_TIME;
 
-    return skytraq_send_message_with_reply(payload, length, reply);
+    return skytraq_send_message_with_reply(payload, length, reply, 100);
```

Comment:
I'm not sure what's happening here. Where does the value of time get updated?

Reply:
It doesn't yet. I did not finish writing out what this should return, hence my question about this in slack